client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/style.scss

```diff
@@ -0,0 +1,281 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+$width-right-panel-card: 40%;
+$width-left-panel-card: calc( 60% - 48px );
+
+.email-provider-stacked-features__whats-included {
+	color: var( --studio-gray-20 );
+	display: block;
+	font-size: 0.75rem;
+}
+
+.email-provider-stacked-features__feature {
+	margin-left: 28px;
+	margin-top: 8px;
+
+	&:first-child {
+		margin-top: 0;
+	}
+
+	&::before {
+		content: '';
+		display: block;
+		height: 1px;
+		margin-bottom: 8px;
+	}
+
+	&:first-child::before {
+		display: none;
+	}
+
+	.gridicon {
+		fill: var( --studio-gray-20 );
+		margin-left: -28px;
+		margin-top: 2px;
+		position: absolute;
```

Comment:
Nit: This won't flow well for responsive displays. Can you find a workaround?

Reply:
Thank you very much Olaseni, I've been able to address it using flex container:
![image](https://user-images.githubusercontent.com/5689927/144291270-5aa0c921-c163-4a03-b85c-47600e458e50.png)